tutorials/.rootlogon.py

```diff
@@ -1,4 +1,4 @@
 """rootlogon module for the Python tutorials. Disables the graphics."""
 import ROOT
 ROOT.gROOT.SetBatch(True)
-ROOT.gROOT.SetWebDisplay("batch")
+ROOT.gROOT.SetWebDisplay("batch7")
```

Comment:
Why is `SetBatch(True)` not sufficient?

Reply:
`SetBatch(True)` force batch mode for old ROOT graphics.
It also automatically set when no `DISPLAY` variable set on Linux.
But it does not mean that web widgets have to be in batch mode.
`SetWebDisplay("batch")` forces all web widgets use only batch mode (image production with headless browser).
`SetWebDisplay("batch7")` enables batch mode only for `RCanvas/RBrowser`, but not for `TCanvas/TBrowser`

Reply:
If a user calls `SetBatch()` the user very likely wants batch mode. What am I missing? I really think we should not add this complexity.